Karma/Karma/SignInForm.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Karma
+{
+    public partial class SignInForm : Form
+    {
+        public SignInForm()
+        {
+            InitializeComponent();
+        }
+
+        private void button3_Click(object sender, EventArgs e)
```

Comment:
The method name should be more clear, name it something like LoginButton_Click or LoginButton_onClick (there should be a discussion about the code format in the future).  Changing the name might be a little bit annoying with winForms, but I think you can change the name with through GUI.